packages/cli-repl/src/cli-repl.ts

```diff
@@ -310,10 +311,14 @@ class CliRepl {
    * The greeting for the shell.
    */
   greet(): void {
-    const { version } = require('../package.json');
+    const { version, engines } = require('../package.json');
     console.log(`Using MongoDB: ${this.internalState.connectionInfo.buildInfo.version}`);
     console.log(`${clr('Using Mongosh Beta', ['bold', 'yellow'])}: ${version}`);
     console.log(`${MONGOSH_WIKI}`);
+    if (semver.lt(process.version, engines.node.replace('^', ''))) {
```

Comment:
semver has a already a feature for that:

```
semver.satisfies('1.2.3', '1.x || >=2.5.0 || 5.0.0 - 7.2.3')
```

Reply:
^ +1

Reply:
Oh great, ty!

Reply:
done

Reply:
It seems satisfies does not support `^` and we'd have to replace the carat anyway

Reply:
Ok ok sorry, it does work, just forgot that the bool is switched. Done!